### PR TITLE
Add the py-coloredlogs package

### PR DESCRIPTION
**Pull request by koning**
This package is a requirement for py-merlinwf

### CODE REVIEW

**Comment by koning** [comment]
This package depends on #13853

File: var/spack/repos/builtin/packages/py-coloredlogs/package.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+# ----------------------------------------------------------------------------
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/py-coloredlogs/package.py]: Can we remove this line?

Comment (koning) [reply var/spack/repos/builtin/packages/py-coloredlogs/package.py]: I removed that line.

Comment (alalazo) [reply var/spack/repos/builtin/packages/py-coloredlogs/package.py]: Thanks!

Comment (alalazo) [reply var/spack/repos/builtin/packages/py-coloredlogs/package.py]: Sorry, I think you removed the line above the one I marked. I know it's a bit of an OCD, but all the other packages don't have it :sweat_smile:

File: var/spack/repos/builtin/packages/py-coloredlogs/package.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyColoredlogs(PythonPackage):
+    """Colored terminal output for Python's logging module"""
+
+    homepage = "https://pypi.org/project/coloredlogs/"
+    url      = "https://files.pythonhosted.org/packages/63/09/1da37a51b232eaf9707919123b2413662e95edd50bace5353a548910eb9d/coloredlogs-10.0.tar.gz"
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-coloredlogs/package.py]:
```suggestion
    url      = "https://pypi.io/packages/source/c/coloredlogs/coloredlogs-10.0.tar.gz"
```

See https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#url

File: var/spack/repos/builtin/packages/py-coloredlogs/package.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+    version('10.0', sha256='b869a2dda3fa88154b9dd850e27828d8755bfab5a838a1c97fbc850c6e377c36')
+
+    depends_on('py-setuptools', type='build')
+    depends_on('py-humanfriendly', type='run')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-coloredlogs/package.py]:
```suggestion
    depends_on('py-humanfriendly@4.7:', type=('build', 'run'))
```

File: var/spack/repos/builtin/packages/py-coloredlogs/package.py (hunk 2 above)
Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-coloredlogs/package.py]: Still need to fix URL

Comment (koning) [reply var/spack/repos/builtin/packages/py-coloredlogs/package.py]: Sorry thought I had done that, but it wasn't pushed.

**Pull request merged**